secure_uninstall/README.md

```diff
@@ -0,0 +1,5 @@
+Secure Uninstall
+================
+
+Ask admin password ('admin_passwd' key from config file)
+before to proceed to module uninstallation
```

Comment:
why not validate the supplied password using the admin password from the database?